WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewController.swift

```diff
@@ -303,6 +324,54 @@ private extension DashboardViewController {
         updatedDashboardUI.displaySyncingError = { [weak self] in
             self?.showTopBannerView()
         }
+
+        // Bottom banner
+        // TODO: 5362 & 5368 - Display banner for JCP sites and if the banner has not been dismissed before.
+        let shouldShowJetpackBenefitsBanner = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.jetpackConnectionPackageSupport)
+        if shouldShowJetpackBenefitsBanner {
+            showJetpackBenefitsBanner(contentView: contentView)
+        } else {
+            hideJetpackBenefitsBanner()
+        }
+    }
+
+    func showJetpackBenefitsBanner(contentView: UIView) {
+        hideJetpackBenefitsBanner()
+        guard let banner = bottomJetpackBenefitsBannerController.view else {
+            return
+        }
+        contentBottomToContainerConstraint?.isActive = false
+
+        addChild(bottomJetpackBenefitsBannerController)
+        containerView.addSubview(banner)
+        bottomJetpackBenefitsBannerController.didMove(toParent: self)
+
+        banner.translatesAutoresizingMaskIntoConstraints = false
+
+        // The banner height is calculated in `viewDidLayoutSubviews` to support rotation.
+        let contentBottomToJetpackBenefitsBannerConstraint = banner.topAnchor.constraint(equalTo: contentView.bottomAnchor)
+        self.contentBottomToJetpackBenefitsBannerConstraint = contentBottomToJetpackBenefitsBannerConstraint
+
+        NSLayoutConstraint.activate([
+            contentBottomToJetpackBenefitsBannerConstraint,
+            banner.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+            banner.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+            // Pins from the safe area layout bottom to accommodate offline banner.
+            banner.safeAreaLayoutGuide.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+        ])
+    }
+
+    func hideJetpackBenefitsBanner() {
+        contentBottomToJetpackBenefitsBannerConstraint?.isActive = false
+        contentBottomToContainerConstraint?.isActive = true
+        if isJetpackBenefitsBannerShown() {
+            bottomJetpackBenefitsBannerController.view?.removeFromSuperview()
+            remove(bottomJetpackBenefitsBannerController)
+        }
+    }
+
+    func isJetpackBenefitsBannerShown() -> Bool {
```

Comment:
nit: What do you think about changing this into a computed property?

Reply:
Makes sense 👍 updated in https://github.com/woocommerce/woocommerce-ios/pull/5380/commits/9d4660d0c4b8b7f9cefcca3eb0e3826998c765c5